src/components/Restaurant.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div>
+    <div class="restaurant" v-if="restaurants.length > 0">
+      <div
+        class="restaurant__img"
+        v-if="restaurant.restaurant.featured_image.length > 0"
+        :style="{'background-image': 'url(' + restaurant.restaurant.featured_image + ')' }"
```

Comment:
Are you sure every featured_image will look good on all screen widths and heights? The image will be cropped (cutted) depending on those parameters.